components/menu-items-separator/menu-items-separator.js

```diff
@@ -0,0 +1,7 @@
+import './style.scss';
```

Comment:
This component is no longer needed. I refactored `<MenuItemsGroup />` component to add this separator using CSS. See: https://github.com/WordPress/gutenberg/pull/5161/files#diff-d3e74855ef057269d48e7afc16600a65R17.